hotkey.js

```diff
@@ -14,10 +14,50 @@ function ignoreHotkeysIn(target) {
   )
 }
 
+function isElementIsIgnoredInRange(parent, node) {
+  if (node.tagName === 'BODY') return null
+  if (parent === node) return null
+  if (node.hasAttribute('data-keyux-ignore-hotkeys')) return node
+
+  return isElementIsIgnoredInRange(parent, node.parentNode)
+}
+
+function getActiveElementInRange(activeElement, elements, container) {
+  for (let element of elements) {
+    let ignoreElement = isElementIsIgnoredInRange(activeElement, element)
+
+    if (ignoreElement) continue
+
+    if (activeElement.hasAttribute('data-keyux-hotkeys')) {
```

Comment:
Why do we call this line on every `elements` if the result is equal for every element?

Reply:
done